docs/src/rsh/errors/index.md

```diff
@@ -1353,9 +1353,21 @@ assert(2 == 2, "5th assertion")
 This error indicates that the depth of recursion for a function call exceeded the limit allowed.
 This issue may indicate that the recursive function does not have a base case.
 
+Example:
+
+```reach
+load: /hs/t/n/Err_RecursionDepthLimit.rsh
+range: 3 - 5
+```
+
 You can fix this issue by re-writing your recursive function into an iterative set of
 statements, e.g. `{!rsh} while` loop.
 
+```reach
+load: /hs/t/y/while_preamble.rsh
```

Comment:
I would prefer to include a working version of `Err_RecursionDepthLimit` or an erroneous version of `while_preamble` so that the files are more directly relatable to one another.